app/src/main/java/ch/hikemate/app/model/authentication/FirebaseAuthRepository.kt

```diff
@@ -87,7 +72,7 @@ class FirebaseAuthRepository : AuthRepository {
             onSuccess(auth.currentUser)
           } else {
             Log.d("SignInButton", "signInWithCredential:failure")
-            onErrorAction(task.exception ?: Exception("Unknown error"))
+            onErrorAction(task.exception ?: Exception())
```

Comment:
Question: Why did you remove the String? Is it bad practice?

Reply:
No, it's just that Sonar was complaining that this string was used 3 times without a constant

Reply:
I see, it shouldn't matter too much anyways.